src/bool.cr

```diff
@@ -63,6 +65,18 @@ struct Bool
     io << to_s
   end
 
+  # Compares this bool against another, according to their underlying value.
+  #
+  # ```
+  # false <=> true  # => -1
+  # true <=> false  # => 1
+  # true <=> true   # => 0
+  # false <=> false # => 0
+  # ```
+  def <=>(other : Bool) : Int32
```

Comment:
Code-oragnization-wise, can you move this up with the other operators?

Reply:
done